l10n.yaml

```diff
@@ -0,0 +1,6 @@
+arb-dir: lib/l10n
```

Comment:
Let's put these somewhere outside of `lib/`, so that it's in a different subdirectory from our main source code.

In zulip-mobile we used to have the translation data inside the main source subdirectory (`src/` there), and it was regularly annoying when grepping the code. If there's even one string that matches whatever you're searching for, you'll get a match in every one of the translations… and because we are fortunate to have people contributing translations for over 50 languages, that can easily fill one's screen. So I was very happy to get them out of there, in zulip/zulip-mobile@1fc26a51224a8f9174758f3d245ac4f813ce85d8.

Perhaps `assets/l10n/`? Or `assets/translations/`?

Reply:
I was just following the guide (their example config used `lib/l10n`) but I understand the argument here. I've also kept django templates out of module subdirectories for this very reason. I'll move to `assets/l10n`